### PR TITLE
 Add verifyCommon method

### LINKED ISSUE #8554: Repeated code mainchain and sidechain CCU verify
### Description

The verify methods of the [MainchainCrossChainUpdate](https://github.com/LiskHQ/lisk-sdk/blob/89e7504ef5eb6183aefe576a93be3d6052e56038/framework/src/modules/interoperability/mainchain/commands/submit_mainchain_cross_chain_update.ts#L57-L121) and [SidechainCrossChainUpdate](https://github.com/LiskHQ/lisk-sdk/blob/89e7504ef5eb6183aefe576a93be3d6052e56038/framework/src/modules/interoperability/sidechain/commands/submit_sidechain_cross_chain_update.ts#L32-L91) commands have repeated code that could be centralized in a single function. This is what already happens for the execute function of the same commands with the [executeCommon](https://github.com/LiskHQ/lisk-sdk/blob/89e7504ef5eb6183aefe576a93be3d6052e56038/framework/src/modules/interoperability/base_cross_chain_update_command.ts#L56) function.

### Which version(s) does this affect? (Environment, OS, etc...)

`v6.0.0-beta.1`

### PR DESCRIPTION
**Pull request by sitetester**
### What was the problem?

This PR resolves #8554 & #8811

### How was it solved?

LIP changes applied

### How was it tested?

Test cases updated

### CODE REVIEW

File: framework/test/unit/modules/interoperability/sidechain/commands/submit_sidechain_cross_chain_update.spec.ts
Comment (AndreasKendziorra) [inline framework/test/unit/modules/interoperability/sidechain/commands/submit_sidechain_cross_chain_update.spec.ts:1]: Shouldn't it be tested that `verifyCommon` is called?

File: framework/test/unit/modules/interoperability/mainchain/commands/submit_mainchain_cross_chain_update.spec.ts
Comment (AndreasKendziorra) [inline framework/test/unit/modules/interoperability/mainchain/commands/submit_mainchain_cross_chain_update.spec.ts:1]: Shouldn't it be tested that `verifyCommon` is called?

**Review by AndreasKendziorra — CHANGES_REQUESTED** [review]

File: framework/test/unit/modules/interoperability/mainchain/commands/submit_mainchain_cross_chain_update.spec.ts [hunk 1]
```diff
@@ -369,85 +386,33 @@ describe('SubmitMainchainCrossChainUpdateCommand', () => {
...
+				});
 		});
 
-		it('should not reject when first CCU contains a certificate older than LIVENESS_LIMIT / 2 when inboxUpdate is empty', async () => {
-			await interopMod.stores.get(ChainAccountStore).set(stateStore, params.sendingChainID, {
-				...partnerChainAccount,
-				status: ChainStatus.REGISTERED,
-			});
-
+		it(`should verify liveness condition when sendingChainAccount.status == ${ChainStatus.REGISTERED} and inboxUpdate is empty`, async () => {
```
[the remark this answers is not in the document]
Comment (AndreasKendziorra) [reply framework/test/unit/modules/interoperability/mainchain/commands/submit_mainchain_cross_chain_update.spec.ts]: It wasn't fixed yet, right?

[the remark this answers is not in the document]
Comment (AndreasKendziorra) [reply framework/test/unit/modules/interoperability/mainchain/commands/submit_mainchain_cross_chain_update.spec.ts]: oh, it is fixed. sorry

File: framework/src/modules/interoperability/base_cross_chain_update_command.ts [hunk 2]
```diff
@@ -54,6 +54,75 @@ export abstract class BaseCrossChainUpdateCommand<
...
+			throw new Error(
+				`Cross-chain updates from chains with status ${ChainStatus.REGISTERED} must contain a non-empty certificate.`,
+			);
+		}
+
+		if (params.certificate.length > 0) {
+			await this.internalMethod.verifyCertificate(context, params, context.header.timestamp);
+		}
+
+		// or validatorsUpdate.bftWeightsUpdateBitmap != EMPTY_BYTES // ???
```
Comment (ishantiw) [inline framework/src/modules/interoperability/base_cross_chain_update_command.ts:110]:
```suggestion
```

**Review by ishantiw — APPROVED** [review]
Adding suggestion but approving

**Pull request merged**